quickwit/quickwit-storage/src/local_file_storage.rs

```diff
@@ -58,7 +58,7 @@ impl fmt::Debug for LocalFileStorage {
 
 impl LocalFileStorage {
     fn full_path(&self, relative_path: &Path) -> crate::StorageResult<PathBuf> {
-        ensure_valid_relative_path(relative_path)?;
+        // ensure_valid_relative_path(relative_path)?;
```

Comment:
I commented it out temporarifly, I still need to find a way around it: for the file source, we don't want this restriction the file must be a subdirectory of the storage's root.

Reply:
I think we want this restriction though. Let's be careful with PR commenting out that kind of stuff. It could easily slip through the CR process.

Reply:
right! I am still in the process of building up my list of questions and this PR is still a draft! I hope I won't leave this kind of things in when moving it out of draft 😄

Reply:
I actually asked this question [on the discord](https://discordapp.com/channels/908281611840282624/908281611840282627/1156577562391236648) because I find that interactive discussions quickly become confusing on Github. But if you prefer, I can centralize all comments here, I don't mind.